### ISSUE TITLE
Missing configuartion varaiable custom_live_log_socket_url

### ISSUE THREAD

**Issue by tigerduck42** [jhuckaby/Cronicle#710]
## Summary
I had issues to get the live log running. So I looked at the code and found a config parameter `custom_live_log_socket_url` which is used in

 https://github.com/jhuckaby/Cronicle/blob/master/htdocs/js/pages/JobDetails.class.js#L869 

but it was never initialised.

See https://github.com/jhuckaby/Cronicle/blob/master/lib/api/config.js#L32-L4

### Operating system and version?
Ubuntu: 20.04

### Node.js version?
v21.6.0

### Cronicle software version?
Cronicle: 0.9.40

### Are you using a multi-server setup, or just a single server?
Single-server

### Are you using the filesystem as back-end storage, or S3/Couchbase?
Filesystem

**Comment by jhuckaby** [comment]
Yeah, this feature never really got implemented.  It was a half-baked attempt at trying to solve log watcher issues people were having, but it died on the vine.  What you are seeing in JobDetails is just leftover, unused code.

This feature was replaced with things like [web_direct_connect](https://github.com/jhuckaby/Cronicle/blob/master/docs/Configuration.md#web_direct_connect), [web_socket_use_hostnames](https://github.com/jhuckaby/Cronicle/blob/master/docs/Configuration.md#web_socket_use_hostnames) and [socket_io_transports](https://github.com/jhuckaby/Cronicle/blob/master/docs/Configuration.md#socket_io_transports).
